DatadogTrace/Sources/OpenTelemetry/OTelNoOpSpan.swift

```diff
@@ -0,0 +1,47 @@
+/*
+* Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+* This product includes software developed at Datadog (https://www.datadoghq.com/).
+* Copyright 2019-Present Datadog, Inc.
+*/
+
+import Foundation
+import OpenTelemetryApi
+
+internal class OTelNoOpSpan: Span {
```

Comment:
suggestion/ Since V2 we [embrace usage of `NOP*` convention](https://datadoghq.atlassian.net/wiki/spaces/RUMP/pages/3027638506/Public+API+Convention+in+iOS+SDK+2.x#Rule-6%3A-Use-of-*Protocol-interface-to-fallback-with-NOP*-instance) instead of `NoOp*`, `Noop*` etc. Let's use it here if possible. Could be worthy adding it [linter rule](https://github.com/DataDog/dd-sdk-ios/blob/5073cf69eb20163af30042a0a9e6e46790526f3b/tools/lint/sources.swiftlint.yml#L75) for this.

Reply:
Sure